models/repo.go

```diff
@@ -1555,7 +1555,7 @@ func GetUserMirrorRepositories(userID int64) ([]*Repository, error) {
 // GetRecentUpdatedRepositories returns the list of repositories that are recently updated.
 func GetRecentUpdatedRepositories(page, pageSize int, userID int64) (repos []*Repository, err error) {
 	repos, _, err = SearchRepositoryByName(&SearchRepoOptions{
-		Keyword:  "%",
+		Keyword:  "",
```

Comment:
I decided to re-enable empty string search for ```GetRecentUpdatedRepositories``` because I couldn't remove the LIKE clause easily without greatly complicating the code in ```SearchRepositoryByName```.  I profiled 3 different versions of the search query on my machine.  
One with LIKE '%%'
One with LIKE '%%%'
and one with the LIKE clause removed and didn't see much of a difference between the 3 queries however I don't have a very large database of repositories either.  If you want I could send you a sql script if you wanted to profile the queries on gogs.io to see if there's any difference.